# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../squashfs-tools/linux-2.6.23.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- squashfs3.2-r2/kernel-patches/linux-2.6.20/squashfs3.2.orig-patch	2007-10-11 17:32:27.000000000 -0300
+++ squashfs3.2-r2/kernel-patches/linux-2.6.20/squashfs3.2-patch	2007-10-11 17:35:23.000000000 -0300
@@ -2399,7 +2399,7 @@
 +	squashfs_inode_cachep = kmem_cache_create("squashfs_inode_cache",
 +	     sizeof(struct squashfs_inode_info),
 +	     0, SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT,
-+	     init_once, NULL);
++	     init_once);
 +	if (squashfs_inode_cachep == NULL)
 +		return -ENOMEM;
 +	return 0;
